ui/tests/unit/adapters/clients-activity-test.js

```diff
@@ -9,10 +9,11 @@ module('Unit | Adapter | clients activity', function (hooks) {
   setupMirage(hooks);
 
   hooks.beforeEach(function () {
+    const date = new Date('2023-01-20');
```

Comment:
When I was pairing with Angel the assumption is that when testing using current day, it may cause issues in testing. You do bring up a good point where it should reliable especially since this how we do it in our app. Looking at the tests more closely it looks like we are subtracting the date by `twoMothsAgo.getMonth() - 2` and this results in a negative number! :D

Reply:
Just pushed up the changes @zofskeez! We were adding/subtracting the months twice in a two different ways so it was causing some bugs. Thank you! :D

Reply:
Great I'm glad you were able to track it down!